### DISCUSSION TITLE
SDK 1.7 Maintenance

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#666] [Q&A] [answered]
We intend to go into Production with SDK 1.7. 

It sounds to me that in the workshops there was talk of a 12-month maintenance period for the sdk. 

Until when could SDK 1.7 be used in production for sending eForms?

Thanks in advance,
Alex

**Comment by ferrakaop** [comment, marked as answer]
Our policy is to support SDK versions for 12 months and we aim announce end of support 6 months before.  

SDK 1.7 was released in May this year so we will support it until May 2024. This means that it remains valid for submission of notices and we also patch the release with the latest view-templates and labels/translations (see 1.7.1 for example).

For info, we have extended support for SDK version 1.3, 1.4 and 1.5 until January 2024 so they have a couple of months longer to allow early adopters the time to upgrade SDK.

For participants in the next workshop, we are collecting information about which SDKs are or will be used: https://op.europa.eu/en/web/ted-eforms/registration

This will give us a better overview to plan when we deprecate SDK versions.

**Comment by AlexAS81** [comment]
Could you tell us the exact date when it can no longer be used?

Thanks in advance,
Alex

**Reply by ferrakaop** [reply]
The last day for submissions in SDK 1.7 has been extened to 30 September 2024.  Please see the recently updated https://docs.ted.europa.eu/home/eforms/active-versions/index.html for more details.  We will also email eSenders to bring this to everyone's attention and we can clarify further at the 20 March workshop https://op.europa.eu/en/web/ted-eforms/home
